frontend/src/screens/learner.tsx

```diff
@@ -191,11 +218,48 @@ export const DesktopStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies }
                 modules={[FreeMode, Pagination, Navigation]}
             >
                 {studies.map(study => (
-                    <SwiperSlide style={{ padding: '1rem' }} key={study.id}>
+                    <SwiperSlide style={{ padding: '1rem', position: 'static'}} key={study.id}>
                         <StudyCard study={study} />
                     </SwiperSlide>
                 ))}
-            </Swiper>
+            </Swiper> */}
+            <Flex ref={(ele) => setElement(ele)} align='center' justify='flex-start' gap='lg' pt='1rem' pb='2rem' style={{ overflowX: 'auto', overflowY: 'hidden',
```

Comment:
I wonder if you could utilize the [mantine scroll-area](https://mantine.dev/core/scroll-area/) for this horizontal scrolling. Some of the styles in this section shouldn't be written in-line too, i think in general we could probably clean up this section a bit, what do you think??